### ISSUE TITLE
Instrumenting fails on variable name `class`

### ISSUE THREAD

**Issue by DeltaBalances** [sc-forks/solidity-coverage#200]
I ran into an issue where my contract compiled, but the instrumenting failed on a syntaxError.
This turned out to be due to a variable named `class` .

A minimal contract that replicates this:
```
contract AA {
    event MyEvent (
        uint256 class
    );
}
```
Which results in
```
Instrumenting  ./coverageEnv/contracts/AA.sol
Cleaning up...
There was a problem instrumenting ./coverageEnv/contracts/AA.sol: SyntaxError: Expected ")", ",", "[", "constant", "indexed", "memory", "storage", comment, end of line, or whitespace but "c" found. Line: 3, Column: 17
Exiting without generating coverage...
```

Renaming it to `klass` or `cass` solves the problem, showing that the problem is with the exact word.

In many languages `class` is a reserved keyword, but as far as I know this isn't the case for solidity. Neither Truffle nor Remix give any warnings about it and the contract itself compiles fine.

The error is trivial to avoid but it got me wondering, is this expected behavior or a bug?

**Comment by cgewecke** [comment]
@DeltaBalances Thank you, yes this is a bug. The parser is adapted from a JS parser and the `class` keyword is incorrectly listed there as a 'future reserved' keyword. The correct list of reserved Solidity keywords is:
```
abstract, after, case, catch, default, final, in, inline, let, match, null, of, 
relocatable, static, switch, try, type, typeof
```

Will fix this over the weekend.

**Comment by cgewecke** [comment]
Thanks for finding this @DeltaBalances. Fix is published w/ `0.4.12`
